build.sh

```diff
@@ -43,3 +43,7 @@ do
   fi
   cd ..
 done
+
+echo "Extracting userscript meta blocks."
+
+extractUserScriptMeta.sh
```

Comment:
If this is in the same directory as build.sh, it'll need a `./` in front of it for the call to work. The file will need to be chmodded for the `x` bit, as well, which will require someone running not-Windows.